src/index.ts

```diff
@@ -323,21 +323,6 @@ Please refer to the Monika documentations on how to how to configure notificatio
   }
 }
 
-// Subscribe FirstEvent
```

Comment:
maybe just keep the listeners but remove the log.info so that in the future we just need to fill the functions.